Jenkinsfile

```diff
@@ -241,11 +241,12 @@ stage('Configure') {
 					new EsLocalBuildEnvironment(version: '7.16.3', condition: TestCondition.ON_DEMAND),
 					new EsLocalBuildEnvironment(version: '7.17.0', condition: TestCondition.AFTER_MERGE),
```

Comment:
Should we bump the `7.17` to `7.17.7` as we run against it after the merge?

Reply:
Right, I'll send another PR.

Reply:
thanks 😃